Library/Formula/img2xterm.rb

```diff
@@ -0,0 +1,16 @@
+class Img2xterm < Formula
+  desc "Display images on terminal"
+  homepage "https://github.com/kfei/img2xterm"
+  head "https://github.com/kfei/img2xterm.git"
+  url "https://github.com/kfei/img2xterm/archive/master.tar.gz"
```

Comment:
We need a versioned tarball, otherwise the SHA won't match as soon as another commit is pushed.  If you can't convince the developers to tag a release, we'll need to send this to our head-only repository instead.